src/app/ReviewService/ReviewPrompter.iOS.cs

```diff
@@ -11,14 +13,37 @@ namespace ReviewService;
 /// </summary>
 public sealed class ReviewPrompter : IReviewPrompter
 {
+	private readonly ILogger _logger;
+
+	public ReviewPrompter(ILogger<ReviewPrompter> logger)
+	{
+		_logger = logger ?? NullLogger<ReviewPrompter>.Instance;
+	}
+
 	/// <inheritdoc/>
 	public Task TryPrompt()
 	{
-		Foundation.NSRunLoop.Main.BeginInvokeOnMainThread(() =>
+		try
+		{
+			_logger.LogDebug("Trying to prompt star rating to user.");
+
+			Foundation.NSRunLoop.Main.BeginInvokeOnMainThread(async () =>
+			{
+				await CrossStoreReview.Current.RequestReview(false);
+			});
+
+			_logger.LogTrace("Prompting star rating to user...");
```

Comment:
May be we should move it just before the `BeginInvokeOnMainThread`? Also just use one dot at the end, looks cleaner in my opinion.

Reply:
We could also say `prompt for a review` instead of prompt star rating? What do you think?

Reply:
Prompting for a review is also valid.

Reply:
I agree on the rephrase but the LogTrace Im thinking maybe we could put it before the await CrossStoreReview call ?

Reply:
Well yes but not in the UI thread call we don't want it do more work than necessary. So before Foundation.X.X

Reply:
Right didn't think of that